ios/build.gradle.kts

```diff
@@ -0,0 +1,50 @@
+plugins {
+    kotlin("multiplatform")
+    kotlin("native.cocoapods")
+    id("org.jetbrains.compose")
+    id("com.google.devtools.ksp")
+}
+
+kotlin {
+
+    iosX64()
+    iosArm64()
+    iosSimulatorArm64()
+
+    cocoapods {
+        version = "1.0.0"
+        summary = "appyx-starter-kit iOS module"
+        ios.deploymentTarget = "16.4"
+        podfile = project.file("../iosApp/Podfile")
+        framework {
+            baseName = "ios"
+            isStatic = true
+        }
+    }
```

Comment:
I tried running the branch and Cocoapods is complaining that is missing the fields for: homepage, author, and license.
I checked how it was on the demo project and it looks like this:
```
cocoapods {
    version = "1.0.0"
    summary = "appyx-starter-kit iOS module"
    homepage = "https://bumble-tech.github.io/appyx/"
    ios.deploymentTarget = "16.4"
    podfile = project.file("../iosApp/Podfile")
    framework {
        baseName = "ios"
        isStatic = true
    }
    license = "Apache License, Version 2.0"
    authors = "https://github.com/bumble-tech/"
}
```

Reply:
Added missing values.